src/core/a-entity.js

```diff
@@ -226,7 +226,6 @@ var proto = Object.create(ANode.prototype, {
   addToParent: {
     value: function () {
       var parentNode = this.parentEl = this.parentNode;
-
```

Comment:
Not a big deal, but I liked to separate declaration with logic using whitespace.